src/Microsoft.OData.Core/ODataBatchOperationRequestMessage.cs

```diff
@@ -57,7 +73,9 @@ public sealed class ODataBatchOperationRequestMessage : IODataRequestMessage, IO
             string contentId,
             IODataPayloadUriConverter payloadUriConverter,
             bool writing,
-            IServiceProvider container)
+            IServiceProvider container,
+            IEnumerable<string> dependsOnIds = null,

```

Comment:
This internal constructor is called in two places, and both provide dependsOnIds, so there is no reason to define a default null value.

Reply:
cleaned up. Thanks.

---
In reply to: [155401617](https://github.com/OData/odata.net/pull/992#discussion_r155401617) [](ancestors = 155401617)